### PR TITLE
Add buttons to Image Toolbar

### PR DESCRIPTION
**Pull request by rogerhutchings**
Package: `lib-classifier`

Adds the buttons to the image toolbar. Requires ~~#80~~ and ~~#81~~ merging first.

Originally from #74 

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

File: packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/FullscreenButton/FullscreenButton.spec.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  it('should call the enableFullscreen prop function on click if fullscreen is not active', function () {
+    const spy = sinon.spy()
+    const wrapper = mount(
+      <FullscreenButton
+        fullscreen={false}
+        enableFullscreen={spy}
+      />
+    )
+    wrapper.find('button').simulate('click')
+    expect(spy.called).to.equal(true)
```
Comment (srallen) [inline packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/FullscreenButton/FullscreenButton.spec.js]: `expect(spy.called).to.be.true` is the preferred BDD syntax with Chai: http://www.chaijs.com/api/bdd/#method_true

File: packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/FullscreenButton/FullscreenButton.spec.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  it('should call the disableFullscreen prop function on click if fullscreen is active', function () {
+    const spy = sinon.spy()
+    const wrapper = mount(
+      <FullscreenButton
+        fullscreen
+        disableFullscreen={spy}
+      />
+    )
+    wrapper.find('button').simulate('click')
+    expect(spy.called).to.equal(true)
```
Comment (srallen) [inline packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/FullscreenButton/FullscreenButton.spec.js]: Same comment about BDD syntax

File: packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/ZoomInButton/ZoomInButton.spec.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import React from 'react'
+import ZoomInButton from './ZoomInButton'
+
+describe('Component > ZoomInButton', function () {
+  it('should render without crashing', function () {
+    shallow(<ZoomInButton />)
+  })
+
+  it('should have an ARIA label', function () {
+    const wrapper = mount(<ZoomInButton />)
```
Comment (srallen) [inline packages/lib-classifier/src/components/Classifier/components/ImageToolbar/components/ZoomInButton/ZoomInButton.spec.js]: Is `mount` needed here for counterpart to run?

**Comment by rogerhutchings** [comment]
I've dismissed @srallen's review, as I've made some changes and she also made some in #101

**Pull request merged**